example/libclang-example/generated_bindings.dart

```diff
@@ -9296,8 +9296,6 @@ typedef NativeClang_Cursor_getTranslationUnit = CXTranslationUnit Function(
 typedef DartClang_Cursor_getTranslationUnit = CXTranslationUnit Function(
     CXCursor arg0);
 
-class CXCursorSetImpl extends ffi.Opaque {}
```

Comment:
This was one of the reasons libclang_example_test.dart was failing, but it looks like a bug to me. `CXCursorSetImpl` is not used anywhere in this file other than `CXCursorSet` below, which has been explicitly overwritten to use `custom_import.CXCursorSetImpl`. So I think there must have been a bug where this binding was included anyway, and I inadvertently fixed it.

Reply:
That's strange. The struct CXCursorSetImpl is included according to the config. So it should've been generated regardless of it's usage.

Is it possible it got excluded because it is also in the type-map config?

Reply:
The current behaviour in type-map is to use the custom-import if it matches, but the actual definitely is still generated if the user includes it in the config.

This might be happening because all Declaration type generations now go through getCodegenType.

@dcharkes if this behaviour is fine, we can perhaps let this be. The catch is that one would need to remove the type-map to generate the original definition.

Reply:
If we have it in the type map, I cannot come up with a reason for people to want to have the generated one as well.l

If there's a use who does, we can always make a patch again.